scalafmt-tests/src/test/scala/org/scalafmt/util/HasTests.scala

```diff
@@ -43,9 +43,6 @@ trait HasTests extends FormatAssertions {
     )
 
   lazy val debugResults = mutable.ArrayBuilder.make[Result]
-  val testDir = new File(
```

Comment:
`ManualTests` and `UnitTests` is failing for me now.

Reply:
This should be fixed now.